sdk/core/src/services/WalletStorageService.ts

```diff
@@ -340,18 +298,54 @@ export default class WalletStorageService {
     return credentials
   }
 
-  async getCredentialByIndex(credentialIndex: number): Promise<any> {
-    const paginationOptions: FetchCredentialsPaginationOptions = { skip: credentialIndex, limit: 1 }
-    const credentials = await this.fetchDecryptedCredentials(paginationOptions)
+  private async _authorizeVcVault(audienceDid: string): Promise<string> {
```

Comment:
Please rename to `authorizeAffinidiVcVault` so that it is clear what methods are for legacy bloom vault and what are for new affinidi vault